lib/fastimage.rb

```diff
@@ -62,1005 +62,28 @@
 require 'base64'
 require 'uri'
 
+require 'fastimage/fastimage_parsing/stream_util'
```

Comment:
I tend to either autoload or require_relative

require certainly works but it has to walk the loadpath.